src/dev/tests/unit/framework/TechDivision/MagentoUnitTesting/TestCase/Abstract.php

```diff
@@ -895,6 +918,27 @@ public function setPrivateProperty($property, $value, $instance = null)
     }
 
     /**
+     * Call private or protected Method
+     *
+     * @param string $method
+     * @param array $params
+     * @param object $instance
+     *
+     * @return mixed
+     */
+    public function callPrivateMethod($method, $params, $instance = null)
```

Comment:
There is already a method "callMethod" with exactly the same functionality. In order to constant naming (e.g. setPrivateProperty) we should implement a method called "callPrivateMethod"... please change your implementation to something like following:

```php
public function callPrivateMethod($methodName, array $args = array(), $instance = null)
{
    return $this->callMethod($methodName, $args, $instance);
}
```